core/templates/components/question-directives/question-misconception-editor/question-misconception-editor.directive.html

```diff
@@ -0,0 +1,28 @@
+<div class="oppia-tagged-misconception-title" ng-if="$ctrl.misconceptionName && !$ctrl.outcome.labelledAsCorrect">
+  <label> Tagged Misconception: </label>
+  <[$ctrl.misconceptionName]>
+</div>
+<button type="button"
+        ng-if="$ctrl.isEditable && $ctrl.rules && !$ctrl.outcome.labelledAsCorrect && $ctrl.containsMisconceptions()"
+        class="btn btn-primary btn-lg oppia-add-rule-button protractor-test-add-answer"
+        style="padding-top: 5px;"
+        ng-click="$ctrl.tagAnswerGroupWithMisconception()">
+  Tag with misconception
+</button>
+<style>
+  .oppia-add-rule-button {
+    background-color: rgba(5,140,166,0.9);
+    border: 0;
+    border-radius: 0;
+    margin-top: 10px;
+    width: 100%;
+  }
+  .oppia-add-rule-button:active,
+  .oppia-add-rule-button:focus,
+  .oppia-add-rule-button:hover {
+    background-color: rgba(5,140,166,1);
+  }
+  .oppia-tagged-misconception-title {
+    margin-top: 10px;
+  }
+</style>
```

Comment:
Missing newline at EOF. @Hudda shouldn't the linter catch this?

Reply:
@seanlip We do not have newline checks yet, already added in #8423

Reply:
Thanks @Hudda!

Just wondering -- is it worth doing some of the "easy" stuff from #8423 quickly (assuming it's just a change to eslint/similar)? If there's any low-hanging fruit, it might be better to knock it off now so that it doesn't keep cropping up.

Reply:
@seanlip eslint does not support this check yet. we have to implement a custom check ([see comment](https://github.com/oppia/oppia/pull/8519#discussion_r410666304))

Reply:
@Hudda Sorry -- what does that comment have to do with "missing newline at EOF"? (Thinking about it, I guess it should be a standard general check that's independent of the filetype.)

Also, can you confirm that everything on #8423 needs a custom implementation -- nothing can be done by just adding a config to pylint/eslint/etc.?

Reply:
@seanlip Ah Sorry -- I misread it.

> Also, can you confirm that everything on #8423 needs a custom implementation

Yep, everything on #8423 needs a custom implementation.

Reply:
Thanks! I added that note to the first comment in #8423.

Reply:
Done